media/css/_variables.scss

```diff
@@ -16,6 +16,7 @@ $light-grayish-blue: #e7ecf2;
 $bright-red: #de2346;
 $grayish-cyan: #88adab;
 $dictionary-inactive: #70706f;
```

Comment:
You'd better move these 2 variable declarations (`$dictionary-inactive` and `$teachers-container`) down to line 29, since they don't really represent color names.